contracts/DepegRiskpool.sol

```diff
@@ -160,6 +160,56 @@ contract DepegRiskpool is
         if(keccak256(abi.encodePacked(name)) != EMPTY_STRING_HASH) {
             _bundleIdForBundleName[name] = bundleId;
         }
+
+        // Register the new bundle with the staking/bundle registry contract. 
+        // Staking and registry are set in tandem (the address of the registry is retrieved from staking),
+        // so if one is present, its safe to assume the other is too.
+        if (address(_bundleRegistry) != address(0)) { 
+            registerBundleInRegistry(bundleId, name, lifetime);
```

Comment:
either here or in registerBundleInRegistry/reigsterBundleForStaking: ensure that riskpool is registered for staking and only proceed with registry if this is the case. otherwise just skip registration of bundle